src/core/thread/mesh_forwarder_ftd.cpp

```diff
@@ -160,6 +160,13 @@ void MeshForwarder::HandleResolved(const Ip6::Address &aEid, Error aError)
 
             if (aError == kErrorNone)
             {
+#if OPENTHREAD_CONFIG_BACKBONE_ROUTER_ENABLE
+                // Pass back to IPv6 layer for DUA destination resolved by Backbone Query
+                if (ForwardDuaToBackboneLink(*cur, ip6Dst) == kErrorNone)
+                {
+                    ExitNow();
```

Comment:
What if there are multiple packets enqueued for the same destination?

Reply:
fixed